packages/bitcore-wallet-client/src/lib/api.ts

```diff
@@ -611,10 +611,21 @@ export class API extends EventEmitter {
     }
     var widHex = Buffer.from(walletId.replace(/-/g, ''), 'hex');
     var widBase58 = new Bitcore.encoding.Base58(widHex).toString();
+    let networkChar = 'L';
+    switch (network) {
+      case 'testnet':
+        networkChar = 'T';
+        break;
+      case 'regtest':
+        networkChar = 'R';
+        break;
+      default:
+        networkChar = 'L';
+    }
```

Comment:
Looks fine. Although it would perhaps be more apparent that you're simply performing a mapping action if you did something like:
const networkToNetworkChar = {
  livenet: 'L',
  regtest: 'R',
  testnet: 'T'
};
const networkChar = networkToNetworkChar[network] || networkToNetworkChar.livenet;

Certainly potato/potato. Same goes for the mapping in the other direction below.